front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -75,7 +75,7 @@ const LinearScaleControl = ({
                 style={{ lineHeight: '0px' }}
               >
                 <Box
-                  mt={isSmallerThanXlPhone ? '0px' : '16px'}
+                  mt={isSmallerThanXlPhone ? '0px' : '0px'}
```

Comment:
If it's always 0px, then I think we can just use `mt="0px" `here :)

Reply:
Ah yes, missed that. Updated!